src/GitVersionExe.Tests/UpdateWixVersionFileTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace GitVersionExe.Tests
```

Comment:
Why only `GitVersionExe` tests and not for `GitVersionCore`? Also, don't you agree approval tests such as this is better suited for file manipulation testing such as this?

https://github.com/GitTools/GitVersion/blob/ee5b77a3439b7e905ca8401c7d8ed15b2495b0d6/src/GitVersionCore.Tests/AssemblyInfoFileUpdaterTests.cs#L26-L40

Reply:
Could you please explain this in detail?

Reply:
Yes, sure. Approval tests allows you to create a file on disk that is what the test is compared against. If the content matches, the test is successful, otherwise it fails and a diff is presented.

The above test will verify the manipulated file content against the following approved text file:

https://github.com/GitTools/GitVersion/blob/ee5b77a3439b7e905ca8401c7d8ed15b2495b0d6/src/GitVersionCore.Tests/Approved/cs/AssemblyInfoFileUpdaterTests.ShouldCreateAssemblyInfoFileWhenNotExistsAndEnsureAssemblyInfo.approved.txt#L1-L13